flow-typed/box-ui-elements.js

```diff
@@ -485,10 +485,10 @@ type Errors = {
 };
 
 type FileAccessStats = {
-    preview_count: number,
-    download_count: number,
-    comment_count: number,
-    edit_count: number,
+    preview_count?: number,
```

Comment:
Part of this commit?

Reply:
Did 2 commits get combined?

Reply:
What the heck. That shouldn't be in there